pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -482,6 +482,30 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				},
 			},
 		},
+
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: DockerBuildCreatorRoleName,
```

Comment:
this defines the roles, but doesn't give them to anyone out of the box, so builds won't be able to be created by default